### PR TITLE
Hide Fields based on Technician Role

### LINKED ISSUE #211: Signs Work Order Details: Hide Fields for Technician Users
For technician users only, hide:
- Printed Date (`field_3252`)
- Created Date (`field_3203`)
- Modified Date (`field_3206`)
- Modified By (`field_3283`)
- Work Type (`field_3215`)
- Work Order ID (`field_3214`)

View: `view_1885`

image

### PR DESCRIPTION
**Pull request by johnclary**
Fixes #211.

### CODE REVIEW

**Review by mddilley — APPROVED** [review]
👍 I read the docs about getUserRoles() - that is a handy function!

**Comment by johnclary** [comment]
Thanks. Going to hold off on deploying until other recent PRs are merged.

**Pull request merged**
